src/Behaviour.cxx

```diff
@@ -5,7 +5,6 @@
  * \date   13/10/2020
  */
 
-#include "MGIS/Behaviour/FiniteStrainBehaviourOptions.hxx"
```

Comment:
Read the error messsage that you sent me privately. You somehow manage to mix mgis@1.1 and mgis@master which explains your errors.

Reply:
Right.